README.md

```diff
@@ -66,18 +66,17 @@ services:
         ports:
             - '3000:3000'
     mesh:
-        image: 0xorg/mesh:0xV3
```

Comment:
We should use a specific version here instead of `0xV3`. Otherwise things will break if we release a new version of Mesh with breaking changes (expected to happen soon). We also need to make sure that the version of Mesh itself matches or is compatible with the version of `mesh-rpc-client` used by Launch Kit Backend.